test/gtest/uct/test_wakeup.cc

```diff
@@ -108,6 +114,10 @@ UCS_TEST_P(test_uct_event_fd, am)
     /* make sure the file descriptor IS signaled */
     ASSERT_EQ(1, poll(&wakeup_fd, 1, 1000*ucs::test_time_multiplier()));
 
+    while (m_am_count < 2) {
```

Comment:
where ? I'm asking since if there a "for" loop over 2, I would prefer to have it #define in order to ovoid errors in the future.